recipes/service.rb

```diff
@@ -29,6 +29,15 @@
   else
     service_name = 'openvpn'
   end
+when 'amazon'
+  if node['platform_version'] >= '2'
+    link "/etc/systemd/system/multi-user.target.wants/openvpn@#{node['openvpn']['type']}.service" do
+      to '/usr/lib/systemd/system/openvpn@.service'
+    end
+    service_name = "openvpn@#{node['openvpn']['type']}.service"
+  else
+    service_name = 'openvpn'
+  end
```

Comment:
🎉

Reply:
It's ugly and it, almost works! Just some annoying docker networking on amazon to work out now